detroit-ui-components/src/overlays/Modal.tsx

```diff
@@ -21,12 +21,19 @@ export interface ModalProps extends Omit<OverlayProps, "children"> {
 }
 
 const ModalHeader = (props: { title: string; uniqueId?: string; className?: string }) => {
+  const modalHeader = useRef<HTMLHeadingElement>(null)
+  useEffect(() => {
```

Comment:
I believe this could be simplified to just useEffect(() => modalHeader?.current?.focus(), []) and still achieve the desired functionality